blocks_vertical/procedures.js

```diff
@@ -232,31 +292,6 @@ Blockly.ScratchBlocks.ProcedureUtils.buildShadowDom_ = function(type) {
 };
 
 /**
- * Reattach a block to the correct input after a mutation, and give the
```

Comment:
this isn't new but just noticed it.  In the buildShadowDom function, does the 'hello world' text ever get shown to the user?

Reply:
Yes, the helloWorld text is shown to the user if they:
- had a procedure call with another block attached to the input.
- updated the procedure declaration, causing the mutation to be reapplied to the call
- removed the attached block, revealing the shadow block behind it